InternetHospitalWebUI/src/app/app.module.ts

```diff
@@ -27,6 +27,7 @@ import { PatientGuard } from './Services/Guards/patient.guard';
 import { DoctorGuard } from './Services/Guards/doctor.guard';
 import { ModeratorGuard } from './Services/Guards/moderator.guard';
 import { AdminGuard } from './Services/Guards/admin.guard';
+// tslint:disable-next-line:max-line-length
```

Comment:
do not use this directive here, add new line

Reply:
[tslint] Line breaks are not allowed in import declaration [import-spacing].